ui/src/app/components/popups.tsx

```diff
@@ -59,8 +59,8 @@ interface WithDynamicPositionProps {
   handleClickOutside?: () => void;
   outsideClickIgnoreClass?: string;
   onClick?: () => void;
+  children?: React.ReactNode;
```

Comment:
This is always called with children, and always passes its remaining props to the wrapped component.

The compiler started complaining, so here is a fix.